src/vm/lib/tests_helper.c

```diff
@@ -0,0 +1,50 @@
+#include "tests_helper.h"
+
+#include <string.h>
+#include <stdlib.h>
+
+
+char* string_slice(const char input_string[], int start, int end) {
```

Comment:
I think I will adopt such code naming convention but with a little modification. As with the string library of the VM I think a better naming would be `str_slice(...)`.